src/components/transformation/create/index.tsx

```diff
@@ -391,12 +403,14 @@ function TransformationCreate() {
                                     id: 'newTransform.button.cta',
                                 })}
                         </LoadingButton>
-                        <Stack>
-                            <Typography
-                                variant="body2"
-                                sx={{ color: 'rgb(150,150,150)' }}
-                            >
-                                <FormattedMessage id="newTransform.instructions" />
+                        <Stack spacing={1}>
+                            <Typography variant="caption">
+                                <Box>
+                                    <FormattedMessage id="newTransform.instructions1" />
+                                </Box>
+                                <Box>
+                                    <FormattedMessage id="newTransform.instructions2" />
+                                </Box>
```

Comment:
Changed the color as it was too hard to read.
Made these be on two lines due to how the words wrapped was very odd.